tools/ci/get_version.py

```
@@ -6,67 +6,69 @@
 
     if github_output:
         with open(github_output, 'a') as f:
             f.write(f"version={version}\n")
-            f.write(f"tag={tag}\n")
 
     if should_push_tag:
-        print(f"Creating and pushing new tag: {tag}")
-        subprocess.run(['git', '-c', 'user.name=GitHub Actions', '-c', 'user.email=actions@github.com', 'tag', tag], check=True)
-        subprocess.run(['git', 'push', 'origin', tag], check=True)
+        print(f"Creating and pushing new tag: {version}")
+        subprocess.run(['git', '-c', 'user.name=GitHub Actions', '-c', 'user.email=actions@github.com', 'tag', version], check=True)
```

Ruff (0.15.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)